### PR TITLE
get_rejection_threshold with augment=False

### PR DESCRIPTION
**Pull request by chapochn**
fixes https://github.com/autoreject/autoreject/issues/276

### CODE REVIEW

**Comment by jasmainak** [comment]
Can you update `whats_new.rst` ?

**Comment by chapochn** [comment]
I tried to update my branch to the current master before adding certain changes in test_autoreject.py and in the whats_new.rst. But I have the impression I have done something awkwardly. Sorry about that. Let me know if there is anything I can or need to fix.
I added a test to check that it indeed works with augment=False. Happy to keep it or remove it.

**Comment by jasmainak** [comment]
the diff looks a combination of your previous PR and this PR. Maybe start on a fresh branch and push it?

```sh
$ git branch -m thresholds_no_loc_backup
$ git checkout master
$ git branch thresholds_no_loc
```

then make the changes and commit, and then

```sh
$ git push -f origin thresholds_no_loc
```

**Comment by chapochn** [comment]
OK I started over. I think it worked now.

File: autoreject/utils.py [hunk 1]
```diff
@@ -43,21 +43,22 @@ def _check_data(epochs, picks, ch_constraint='data_channels',
...
 
-    if not _check_ch_locs(picked_info['chs']):
+    if check_loc and not _check_ch_locs(picked_info['chs']):
         raise RuntimeError('Valid channel positions are needed '
                            'for autoreject to work')
 
     # XXX : ch_constraint -> allow_many_types=True | False
     if ch_constraint == 'data_channels':
-        if not all(ch in ('mag', 'grad', 'eeg', 'hbo', 'hbr')
+        if not all(ch in ('mag', 'grad', 'eeg', 'hbo', 'hbr', 'ecog', 'seeg')
```
Comment (jasmainak) [inline autoreject/utils.py:52]: didn't you add these in the last PR?

File: autoreject/utils.py [hunk 2]
```diff
@@ -43,21 +43,22 @@ def _check_data(epochs, picks, ch_constraint='data_channels',
...
-            raise ValueError('AutoReject only supports mag, grad, and eeg '
-                             'at this point.')
+            raise ValueError('AutoReject only supports mag, grad, eeg, and '
+                             'ecog at this point.')
     elif ch_constraint == 'single_channel_type':
         if sum(ch in ch_types_picked for ch in ('mag', 'grad', 'eeg',
-                                                'hbo', 'hbr')) > 1:
-            raise ValueError('AutoReject only supports mag, grad, and eeg '
-                             'at this point.')  # XXX: to check
+                                                'hbo', 'hbr', 'ecog', 'seeg'))\
```
Comment (jasmainak) [inline autoreject/utils.py:58]: backslashes are really ugly ... can you avoid it with proper use of paranthesis?

File: autoreject/utils.py (hunk 1 above)
Comment (chapochn) [reply autoreject/utils.py:52]: `_check_data` is not used in `get_rejection_threshold`, so this utils.py file was not changed before

File: autoreject/utils.py (hunk 2 above)
Comment (chapochn) [reply autoreject/utils.py]: updated

**Comment by jasmainak** [comment]
Looks good, thanks @chapochn ! 🥳

**Comment by chapochn** [comment]
Of course! Glad it works!

**Pull request merged**